docs/_posts/2019-09-15-seminar-meeting-minutes-3-6.md

```diff
@@ -0,0 +1,254 @@
+
+  [Multi-task Learning of Hierarchical Vision-Language Representation](https://arxiv.org/pdf/1812.00500.pdf)
+
+  ![刘云]({{ '/assets/2019/seminar-3-6/seminar-3-6-liuyun.jpg'| relative_url }})
+
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-6/1812-00500-multi-task.pdf' | relative_url }}'
+      width='560'
+      height='315'
+      allowfullscreen
+      webkitallowfullscreen
+      frameborder="0"
+      style="
+        position: absolute;
+        top:0;
+        left:0;
+        width:100%;
+        height:100%;
+      "
+    ></iframe>
+  </div>
+
+- 李卓桓 @[huan](https://github.com/huan)
+  
+  >TPU 的应用案例: [使用 TPU 训练 TensorFlow 模型（Huan）](https://www.zixia.net/tensorflow-handbook-tpu/)，帮助快速进行大规模训练
+  >可能可以申请到2500刀的资源。
+
+  ![李卓桓]({{ '/assets/2019/seminar-3-6/seminar-3-6-huan.jpg'| relative_url }})
+
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-6/tensorflow-handbook-tpu.pdf' | relative_url }}'
+      width='560'
+      height='315'
+      allowfullscreen
+      webkitallowfullscreen
+      frameborder="0"
+      style="
+        position: absolute;
+        top:0;
+        left:0;
+        width:100%;
+        height:100%;
+      "
+    ></iframe>
+  </div>
+
+- 张璐 @[23LuZ](https://github.com/23LuZ)
+  >语言中的注意力机制
+  >Encoder-Decoder 结构
+  >Local/Global/Global-Local 三种分类
+
+  [Neural Responding Machine for Short-Text Conversation](https://www.aclweb.org/anthology/P15-1152)
+
+  ![张璐]({{ '/assets/2019/seminar-3-6/seminar-3-6-23luz.jpg'| relative_url }})
+
+  <div class="zoom-container" style="
+      position: relative;
+      padding-bottom:56.25%;
+      padding-top:30px;
+      height:0;
+      overflow:hidden;
+  ">
+    <iframe
+      src='{{ '/assets/js/viewer-js/#/assets/2019/seminar-3-6/neural-responding-machine-for-short-text-conversation.pdf' | relative_url }}'
+      width='560'
+      height='315'
+      allowfullscreen
+      webkitallowfullscreen
+      frameborder="0"
+      style="
+        position: absolute;
+        top:0;
+        left:0;
+        width:100%;
+        height:100%;
+      "
+    ></iframe>
+  </div>
+
+## 授蛋仪式
+
+CV 授蛋仪式
+![授蛋]({{ '/assets/2019/seminar-3-6/seminar-3-6-cv-chairman-appointment.jpg'| relative_url }})
+
+NLP 授蛋仪式
+![授蛋]({{ '/assets/2019/seminar-3-6/seminar-3-6-nlp-chairman-appointment.jpg'| relative_url }})
+
+## 集体合照
+
+![合照]({{ '/assets/2019/seminar-3-6/seminar-3-6-group-photo.jpg' | relative_url }})
+
+## 俱乐部讨论内容
+
+1. 惯性的时间控制需要每一届主席持续把控
+2. 如果预期讨论过多的poster可以当成oral申报，poster主要还是集中在论文的某一个点讲清楚。
+3. Zoom链接和视频录制正式上线，[入口地址:https://ai-ml.club/zoom/](https://ai-ml.club/zoom/)。
+4. 下一任主席为陈光，下一任副主席为段清华。
```

Comment:
陈光？段清华？

We should announce the chair and vice-chair in our meeting minutes.

It would be great to add the following information to our minutes:

1. AMC(CV) Chairs:
    1. Chair: A
    1. Vice-chair: B
1. AMC(NLP) Chairs:
    1. Chair: 李卓桓
    1. Vice-chair: 张璐